.github/workflows/pcb.yaml

```
@@ -0,0 +1,288 @@
+          name: CHANGELOG
+          path: .
+
+      - name: Download build artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: ${{ env.kibot_variant }}
+          path: .
+
+      - name: ZIP artifact
+        shell: bash
+        run: |
```

actionlint (1.7.8)

shellcheck reported issue in this script: SC1009:info:3:1: The mentioned syntax error was in this simple command

(shellcheck)